tests/pool_test.go

```diff
@@ -115,7 +115,7 @@ var _ = Describe("[sig-compute]VirtualMachinePool", func() {
 
 	newVirtualMachinePool := func() *poolv1.VirtualMachinePool {
 		By("Create a new VirtualMachinePool")
-		pool := newPoolFromVMI(libvmi.NewCirros())
+		pool := newPoolFromVMI(libvmi.New(libvmi.WithResourceMemory("1Mi")))
```

Comment:
I think this is not enough. Depending on the firmware you need 32/132Mi at least.

Reply:
@xpivarc I've been testing it locally and seems like the VM managed to launch and VMI status was running. Would you like me to do some double check with QEMU minimal RAM requirements to launch SeaBIOS?

Reply:
Confirmed that minimum RAM required for seaBIOS is 1M, increased to 2Mi to be on the safe side. Please see updated PR  description

Reply:
Thanks Igor. My numbers were based on the assumption we want to boot the VMs but that is not a goal of these tests.